.github/workflows/deploy-dev-manuelt.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: dev-fss
           RESOURCE: deploy/nais.yaml
-          VAR: APP_NAME=melosys-eessi,VAULT_CLUSTER=dev,INGRESS=https://melosys-eessi.dev.intern.nav.no,KAFKA_POOL=nav-dev,APP_NAME_MELOSYS_API=melosys,TENANT_URL=trygdeetaten.no,GROUP_MELOSYS_INNLOGGING=52981246-8032-40f0-863a-2bd2844f2d57,APP_NAME_EUX_RINA_API=eux-rina-api-q2,CLUSTER_NAME_EUX_RINA_API=dev-fss
+          VAR: APP_NAME=melosys-eessi,VAULT_CLUSTER=dev,INGRESS=https://melosys-eessi.dev.intern.nav.no,KAFKA_POOL=nav-dev,APP_NAME_MELOSYS_API=melosys,TENANT_URL=trygdeetaten.no,GROUP_MELOSYS_INNLOGGING=52981246-8032-40f0-863a-2bd2844f2d57,APP_NAME_EUX_RINA_API=eux-rina-api-q2,CLUSTER_NAME_EUX_RINA_API=dev-fss,APP_NAME_PDL_WEB=pdl-web,CLUSTER_NAME_PDL_WEB=dev-fss,PDL_WEB_URL=https://pdl-web.dev.intern.nav.no
```

Comment:
Tenker at vi ønsker å bruke Vault for den slags, iaf. for PDL-Web URL

Reply:
Ja, tror vi må rydde litt opp i det. Vi er ikke konsekvent med det, så tror vi må få noen PRs på det overalt, ikke bare eessi. Kan fjerne de nye i det minste.